dts/riscv/it8xxx2-wuc-map.dtsi

```diff
@@ -0,0 +1,377 @@
+/*
+ * Copyright 2022 ITE Corporation
```

Comment:
why is this file at the root of `dts/riscv`?

Reply:
Done, move this file to dts/riscv/ite/